frontend/src/components/common/Status/TStatus.vue

```diff
@@ -44,7 +44,11 @@ export default {
                 iconTypeValue: "check-in-circle-classic",
                 iconColor: "#69C297",
               };
-
+            case "Ready":
```

Comment:
I wonder if we should use the constants we have defined here `constants/index.ts`.